src/sessionManager.ts

```diff
@@ -33,16 +33,7 @@ export default function SessionManager(
 
     this.initialize = function (): void {
         if (mpInstance._Store.sessionId) {
-            const sessionTimeoutInMilliseconds: number =
-                mpInstance._Store.SDKConfig.sessionTimeout * 60000;
-
-            if (
-                new Date() >
-                new Date(
-                    mpInstance._Store.dateLastEventSent.getTime() +
-                        sessionTimeoutInMilliseconds
-                )
-            ) {
+            if (hasSessionTimedOut(mpInstance._Store.dateLastEventSent.getTime())) {
```

Comment:
If you follow my suggestion below, this can be destructured and cleaned up.

```suggestion

            const { dateLastEventSent, SDKConfig } = mpInstance._Store;
            const { sessionTimeout } = SDKConfig;
            if (hasSessionTimedOut(dateLastEventSent.getTime(), sessionTimeout)) {
```